components/bookmarks/__tests__/bookmarked-providers-container.test.tsx

```
@@ -0,0 +1,124 @@
+}));
+
+// Mock the BookmarkedProvidersList component
+jest.mock('../bookmarked-providers-list', () => ({
+  BookmarkedProvidersList: ({ 
+    providers, 
+    isLoading, 
+    error, 
+    onRetry, 
+    onRemoveBookmark, 
+    isRemoving 
+  }: any) => (
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)